tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Kafka/KafkaHelper.cs

```diff
@@ -159,6 +176,18 @@ internal static Scope CreateProducerScope(Tracer tracer, ITopicPartition topicPa
                 span.SetTag(Tags.Measured, "1");
 
                 tags.SetAnalyticsSampleRate(KafkaConstants.IntegrationId, tracer.Settings, enabledWithGlobalSetting: false);
+
+                if (dataStreamsManager.IsEnabled)
+                {
+                    span.Context.MergePathwayContext(pathwayContext);
+
+                    // TODO: we could cache this list in a thread local similar to StringBuilderCache to reduce allocations
+                    var edgeTags = string.IsNullOrEmpty(topic)
+                                       ? new[] { $"group:{groupId}", "type:kafka", }
```

Comment:
just a note: @hokitam recently added the idea of hashable tags, and non hashable tags (https://github.com/DataDog/dd-trace-java/pull/3752)

But we can add it later to the .NET tracer depending on how widely it is used in the UI.